Blueprints/BlueprintDefinitions/vs2022/NativeAOTFunction/template/src/BlueprintBaseName.1/aws-lambda-tools-defaults.json

```diff
@@ -8,9 +8,9 @@
   "profile": "DefaultProfile",
   "region": "DefaultRegion",
   "configuration": "Release",
-  "function-runtime": "provided.al2",
-  "function-memory-size": 256,
+  "function-runtime": "dotnet8",
+  "function-memory-size": 512,
   "function-timeout": 30,
-  "function-handler": "bootstrap",
+  "function-handler": "BlueprintBaseName.1",
```

Comment:
Checking my understanding: is `bootstrap` no longer required? Or will the template set it to that anyway?

Reply:
Correct because we are now deploying to `dotnet8` instead of `provided.al2023`. Only in the provided runtimes do you have to name your assembly `bootstrap` and use that as a function handler. A benefit of deploying to `dotnet8` is we don't have to ask users to name their assemblies `bootstrap` and just have to use the name of the assembly for the function handler. You can see in the project file update above I removed the `<AssemblyName>bootstrap</AssemblyName>` property.